xskillscore/core/probabilistic.py

```diff
@@ -621,32 +588,29 @@ def rps(
     ...                          coords=[('x', np.arange(3)),
     ...                                  ('y', np.arange(3)),
     ...                                  ('member', np.arange(3))])
-    >>> category_edges = np.array([.0, .5, 1.])
+    >>> category_edges = np.array([.33, .66])
     >>> xs.rps(observations, forecasts, category_edges, dim='x')
     <xarray.DataArray (y: 3)>
     array([0.14814815, 0.7037037 , 1.51851852])
     Coordinates:
       * y                           (y) int64 0 1 2
-        forecasts_category_edge     <U38 '[0.0, 0.33), [0.33, 0.66), [0.66, 1.0]'
-        observations_category_edge  <U38 '[0.0, 0.33), [0.33, 0.66), [0.66, 1.0]'
+        forecasts_category_edge     <U38 '[-np.inf, 0.33), [0.33, 0.66), [0.66, np.inf]'
+        observations_category_edge  <U38 '[-np.inf, 0.33), [0.33, 0.66), [0.66, np.inf]'
```

Comment:
I'm wondering now if these might be confusing. As in the docstring, the cdf bins are actually all bounded on the left by `-np.inf`, but these are correct if one is thinking about the pdf. I'm not sure what's clearest for the user:

- `'[-np.inf, 0.33), [0.33, 0.66), [0.66, np.inf]'`; or
- `'[-np.inf, 0.33), [-np.inf, 0.66), [-np.inf, np.inf]'`

Happy for you to decide what you think is the clearest, but we should probably be consistent between the docstring and `_assign_rps_category_bounds`

Reply:
Dos the question is whether we think in cdf or pdf? Cumulative bins or single size bins? Although rps is computed based on cdfs, I would still call it category_edges and therefore use the first choice of category edges as coords.